packages/plugins/src/foundry/index.ts

```diff
@@ -76,6 +76,11 @@ const command = args[0]
         'foundry',
         cre
       )
+
+      if (config === false) {
```

Comment:
It'd be nice if we handled this case inside of `readValidatedChugSplashConfig`. For example, we could `process.exit()`. That way, we don't need to always remember to handle this case, and we can get rid of all of these if-statements.